### ISSUE TITLE
Request: Tutorial – General approach to solving a 'screen' with Reactive Cocoa

### ISSUE THREAD

**Issue by bobspryn** [ReactiveCocoa/ReactiveCocoa#854]
Loving Reactive Cocoa so far. To me the most challenging portion is looking at a 'screen' from a storyboard/design and saying, "Ok, so how do I determine what signals make the most sense here and will drive this screen".

Looking for a kind of general approach like:
1. Identify the various configurations the screen can be in
2. Identify what 'states' or circumstances cause those configurations to happen

So basically a "Here's how you might take something from design through to implementation" tutorial would be great, but woven with a 'general approach'. Even the exact implementation is less critical than identifying the general design of your signal logic, but it would be a nice add.
